source/index.html

```diff
@@ -2,11 +2,132 @@
+    <main class="page-main">
+      <h1 class="visually-hidden">Добро пожаловать в Седону</h1>
+      <!-- Вставляю svg с разными шрифтами, а при стилизации с пом background-image
+       дорисовываю фотографию с горным ландшафтом? Логотип - это все надписи или только sedona?
+       На других стр сохраняется только это одно слово. Или это вообще не логотип, а просто
+       другая часть шапки, ведь в шапке сверху уже есть треугольный логотип?
+       И офтоп, но этот вопрос еще всплывет, забывала задать: чем неразрывный пробел лучше
+       переноса строки <br> (и лучше ли)? -->
+      <img src="https://via.placeholder.com/150x150" width="282" height="214" alt="Добро пожаловать в Седону">
+      <section class="advantages">
+        <h2 class="advantages__header">Седона &ndash; небольшой городок в Аризоне, заслуживающий большего!</h2>
+        <p class="advantages__text">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+        <!-- Разметила так, но можно же и просто как отдельные article или списком их сделать?
+             Список внутрь dl нельзя, только что увидела, исправлю -->
+        <dl class="advantages__list">
+          <div class="advantage__item">
+            <dt class="advantage__name">Настоящий городок</dt>
+            <p class="advantage__number">&ndash; 1 &ndash;</p>
+            <dd class="advantage__description">Седона не аттракцион для туристов, там течет своя жизнь</dd>
+            <!-- Имеет смысл к таким фото писать альтернативное описание? На
+               всякий случай написала, вроде в статье с первого курса такое было -->
+            <img src="" width="" height="" alt="Фотография двухэтажного мотеля с флагами на фоне гор">
+          </div>
+            <ul class="advantage__additional-list">
+              <li class="advantage__additional-item">
+                <h3 class="advantage__additional-header">Жилье</h3>
+                <p class="advantage__additional-text">Рекомендуем пожить в настоящем мотеле, все как в кино!</p>
+              </li>
+              <li class="advantage__additional-item">
+                <h3 class="advantage__additional-header">Сувениры</h3>
+                <p class="advantage__additional-text">Не только китайского, но и местного производства!</p>
+              </li>
+              <li class="advantage__additional-item">
+                <h3 class="advantage__additional-header">Еда</h3>
+                <p class="advantage__additional-text">Всегда заказывайте фирменный бургер, вы не разочаруетесь!</p>
+              </li>
+            </ul>
+          <div class="advantage__item">
+            <dt class="advantage__name">Там есть мост дьявола!</dt>
+            <p class="advantage__number">&ndash; 2 &ndash;</p>
+            <dd class="advantage__description">Да, по нему можно пройти! Если вы осмелитесь, конечно</dd>
+            <img src="" width="" height="" alt="Каменный моста дьявола">
+          </div>
+          <div class="advantage__item">
```

Comment:
А вот тут 3 последних `advantages__item` отличаются от предыдущих и тем что фон другой, и тем то картинок нет, и размер другой, соответственно.

Т.е. тут нужно какой-то модификатор (или там, наоборот)

Т.е. класс то же, но ему добавляется модификация:
```suggestion
          <div class="advantages__item advantages__item_secondary">
```

Или наоборот те с картинками, назвать `advantages__item advantages__item_gallery`, а эти серые будут просто `advantages__item`

И так и так правильно. Выбирай тот вариант, который тебе нравится больше всего и больше всего для тебя логичен.

Reply:
Обрати внимание, когда у элемента появляется модификатор надо указывать обязательно и основной класс, и рядом тот же класс с модификатором.